<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>428</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Simulation Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,1,4,2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Choose model</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Ridge</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>LSTM</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,1,1">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>No days to predict</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="number_days">
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>20</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>30</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>60</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>90</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>180</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>210</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Prediction summary</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="styleSheet">
              <string notr="true">color: #000; font-size: 10pt; background-color: transparent; border: none;</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,3">
             <item>
              <widget class="QPushButton" name="start_prediction">
               <property name="text">
                <string>Predict Weather</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reset_prediction">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="weather_prediction_box">
        <property name="title">
         <string>Predictions</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QGridLayout" name="weather_prediction_grid">
           <item row="0" column="0">
            <widget class="QWidget" name="weather_prediction" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
